src/main/java/seedu/address/model/patient/Patient.java

```diff
@@ -17,63 +17,82 @@
 public class Patient {
 
     // Identity fields
+    private final PatientHospitalId patientHospitalId;
     private final Name name;
-    private final Phone phone;
-    private final Email email;
+    private final PreferredName preferredName;
 
     // Data fields
-    private final Address address;
+    private final FoodPreference foodPreference;
+    private final FamilyCondition familyCondition;
+    private final Hobby hobby;
```

Comment:
I think it may make more sense for these fields to be a `Set` as well... Since a person can have multiply food preferences, hobbies and family conditions.

Reply:
After discussion, a separate Issue and PR will be created to handle this enhancement.